src/app/shared/quality-list/quality-list.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+import { Die } from '../die.model';
+
+@Component({
+    selector: 'app-quality-list',
+    templateUrl: './quality-list.component.html',
+    styleUrls: ['./quality-list.component.css'],
+})
+export class QualityListComponent implements OnInit, OnChanges {
+    options: string[];
+    @Input() jsonData: string[] = [];
+    @Input() assignedDie: Die;
+
+    constructor() {}
+
+    ngOnInit(): void {
+        this.options = [];
+    }
+
+    ngOnChanges(): void {
+        this.populateOptionsList();
+    }
+
+    populateOptionsList() {
+        this.options = [];
+        for (let optionName of this.jsonData) {
+            this.options.push(optionName);
+            switch (optionName) {
+            case 'any Mental':
```

Comment:
this area is one that we should discuss in our next meeting, there are some things I think we could reconsider here

Reply:
Yes please.  This was also something else I wanted to discuss.  I'm not happy with how this is implemented at all, but I just wanted something that proved my code worked.  I have a few ideas about how to do this as well, I wonder if we're on the same page.  There's another complication that gets thrown into the quality list selection that I want to bring up as well, since that is going to come up again during other steps.